src/nvlink/wrapper_32bit.py

```
@@ -0,0 +1,583 @@
+            Result code (0 = success, non-zero = error)
+
+        Raises:
+            NVLinkError: If service key setting fails
+        """
+        try:
+            import subprocess
+            import time
+
+            # Set service key in Windows registry
+            reg_key = r"HKLM\SOFTWARE\NAR\NVDTLabLib"
+            result = subprocess.run(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)